ONSdigital/eq-survey-runner
# Replacing User Journey Manager with Questionnaire Manager

### What is the context of this PR?

In our progression towards a simpler code base we're refactoring the user journey manager and combining it with the questionnaire manager. 

I've also introduced an optimisation that means the schema is only parsed once per session and then pickled to the database alongside the user's state.
### How to review
1. Check out this branch
2. Confirm the test pass
3. Check the star wars and MCI surveys and make sure nothing it broken
### Who worked on the PR

@warrenbailey
